demos/sev-demo/README.md

```diff
@@ -0,0 +1,192 @@
+# CCv0 with AMD SEV
+
+CCv0 can be used with AMD SEV. Check out this [demo video](https://youtu.be/SXZ-hzhjahU) to see it in action. This guide will explain how to take an SEV node and use CCv0 with it. This guide does not cover configuring an SEV machine. You will need to have a node where you can start SEV VMs. This guide also does not cover setting up CCv0. You will need a working deployment of CCv0. You will update this deployment to use SEV. 
+Soon many of these steps will be automated via [Kata Deploy](https://github.com/kata-containers/kata-containers/tree/main/tools/packaging/kata-deploy) and the [Confidential Containers Operator](https://github.com/confidential-containers/operator).
+
+## OVMF
+
+You will need to use OVMF. Furthemore, you will need to use the AMDSEV build of OVMF. An appropriate firmware binary is available in this directory.
+The binary in this directory is built from commit `f0f3f5aae7c4d346ea5e24970936d80dc5b60657` and has md5 checksum `bc40de8b6e743e03b3912de64e7d4884`. Any release after [edk2-stable202108](https://github.com/tianocore/edk2/releases/tag/edk2-stable202108) should work. 
+
+You can also build from source. First clone the source and checkout the submodules.
+```
+git clone https://github.com/tianocore/edk2.git
+cd edk2 
+git submodule init
+git submodule update
+```
+Setup the build tools.
+```
+make -C BaseTools/
+source edksetup.sh
+```
+Currently the AMDSEV firmware expects to be built with Grub for reasons unrelated to Confidential Containers. You will need to make a slight modification to the AMDSEV DSC file located at `OvmfPkg/AmdSev/AmdSevX64.dsc`. Commenting the prebuild line will skip trying to build Grub when you build OVMF.
+```
+diff --git a/OvmfPkg/AmdSev/AmdSevX64.dsc b/OvmfPkg/AmdSev/AmdSevX64.dsc
+index 5ee5445116..0657fd7446 100644
+--- a/OvmfPkg/AmdSev/AmdSevX64.dsc
++++ b/OvmfPkg/AmdSev/AmdSevX64.dsc
+@@ -25,7 +25,7 @@
+   BUILD_TARGETS                  = NOOPT|DEBUG|RELEASE
+   SKUID_IDENTIFIER               = DEFAULT
+   FLASH_DEFINITION               = OvmfPkg/AmdSev/AmdSevX64.fdf
+-  PREBUILD                       = sh OvmfPkg/AmdSev/Grub/grub.sh
++#  PREBUILD                       = sh OvmfPkg/AmdSev/Grub/grub.sh^M
+
+   #
+   # Defines for default states.  These can be changed on the command line.
+```
+Since you aren't building Grub, you will also need to create a dummy Grub file located at `OvmfPkg/AmdSev/Grub/grub.efi`
+```
+touch OvmfPkg/AmdSev/Grub/grub.efi
+```
+Now you can build the AMDSEV firmware. The output binary will be located at `Build/AmdSev/DEBUG_GCC5/FV/OVMF.fd`.
+```
+build -t GCC5 -a X64 -p OvmfPkg/AmdSev/AmdSevX64.dsc
+```
+
+Update the Kata configuration file to point to your new firmware.
```

Comment:
Do we need anymore details or example snippet here?

Reply:
I can add a snippet although it's mostly trivial.